docs/cli/platform/config.md

```diff
@@ -14,12 +14,6 @@
 
 * The key of the configuration element to get or set
 
-**Options**
-
-`--value/-v`
```

Comment:
This one is special because is a positional optional argument of the command.

We should not remove the doc for the value option all together.

Rather could you please update command syntax properly in this page (i.e `ern platform config <key> [value]`) and move `value` in arguments. But instead of `--value/-v` it should be `[value]`).

This way we don't lose the description in the doc of what is the value  and the syntax is the proper one.